### ISSUE TITLE
Conflicting cops - Layout/TrailingWhitespace and Layout/EmptyLineAfterGuardClause

### ISSUE THREAD

**Issue by andreheijstek** [rubocop-hq/rubocop#7711]
I have the following code (boilerplate database cleaning setup)

```
  config.before(:suite) do
    if config.use_transactional_fixtures?
      raise(<<-MSG)
        Delete line `config.use_transactional_fixtures = true` from rails_helper.rb
        (or set it to false) to prevent uncommitted transactions being used in
        JavaScript-dependent specs.

        During testing, the app-under-test that the browser driver connects to
        uses a different database connection to the database connection used by
        the spec. The app's database connection would not be able to access
        uncommitted transaction data setup over the spec's database connection.
      MSG
    end
    
    DatabaseCleaner.clean_with(:truncation)
  end
```

If I have a blank line before `DatabaseCleaner.clean_with(:truncation)`
then I get a Layout/TrailingWhitespace warning.
If I remove the blank line
then I get a Layout/EmptyLineAfterGuardClause warning

`rubocop -V`
`0.79.0 (using Parser 2.7.0.2, running on ruby 2.6.5 x86_64-darwin19)`

**Comment by Drenmi** [comment]
Hi, @andreheijstek! 👋 

RuboCop is complaining, not about the line itself, but about the fact that the line has some spaces in it.

<img width="500" alt="Screenshot 2020-02-13 at 3 42 30 PM" src="https://user-images.githubusercontent.com/5259935/74412004-7806ae80-4e77-11ea-9c84-4f4017097e4d.png">

Remove those and you should be good to go. 👍 

(Most editors have a setting that will trim trailing whitespace on save.)
